crates/toolchain/edsl-deserialize/src/lib.rs

```diff
@@ -0,0 +1,8 @@
+mod castf_extension;
+pub mod config;
+pub mod deserialize_instruction;
+
+const LONG_FORM_INSTRUCTION_INDICATOR: u32 = (1 << 31) + 115115115;
```

Comment:
This should be some RISC-V instruction with a 6-bit opcode that's one of the designated custom opcodes